### DISCUSSION TITLE
Bearer token standard

### DISCUSSION THREAD

**Discussion by alanzheng88** [encode/django-rest-framework#9073] [Question & Answer] [answered]
https://github.com/encode/django-rest-framework/blob/7658ffd71d4ad07ddada20b2b9538b889ec02403/docs/api-guide/testing.md?plain=1#L161

I noticed that we're using `Token <api_token>` format here but wouldn't the standard for this be `Bearer <api_token>` or is this working as designed?

in postman we would generally pass a header like `--header 'Authorization: Bearer 1234abcd'`

**Comment by kevin-brown** [comment, marked as answer]
I believe `Bearer` is intended for use as a part of the OAuth Bearer token standard, of which the built-in DRF Token is not. We are allowed to make use of the authorization scheme of our choice and as such we picked `Token`.

**Comment by Naman-12911** [comment]
Hello, 
Can I fix this issue?.
